src/components/atoms/Button/index.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
-import styled from "styled-components";
 import PropTypes from "prop-types";
+
+import styled from "styled-components";
+
```

Comment:
```suggestion
import React from "react";

import PropTypes from "prop-types";
import styled from "styled-components";

```
eslint rule로 보면 styled-component와 prop-types는 개행없이 붙어 있어야 하고 react 밑에 개행이 있어야할 것 같은데 확인 부탁드립니다🙇‍♀️

Reply:
넵 수정하겠습니다